schema.sql

```diff
@@ -9,3 +9,30 @@ CREATE TABLE animals (
 
 /* SECOND SECTION */
 ALTER TABLE animals ADD species varchar;
+
+
+/* THIRD SECTION */
+
+/* Create owners table */
+CREATE TABLE owners(
+id INT GENERATED ALWAYS AS IDENTITY,
+full_name VARCHAR(255),
+age INT,
+PRIMARY KEY (id)
+);
+/* -------------- */
+
+/* Create species table */
+# CREATE TABLE species(
```

Comment:
- [ ] you commented out this code seemingly by accident.

Reply:
Thank you @mikethreels! Fixed! ✔️